OpenChange/MAPIStoreMailMessage.m

```diff
@@ -312,7 +306,19 @@ - (void) _fetchHeaderData
           partParameters = [partHeaderData objectForKey: @"parameterList"];
           encoding = [partHeaderData objectForKey: @"encoding"];
           charset = [partParameters objectForKey: @"charset"];
-          mimeType = [[keys objectAtIndex: i] objectForKey: @"mimeType"];  
+          mimeType = [bodyStructureKey objectForKey: @"mimeType"];
+
+          /* multipart/mixed is the default type.
+             multipart/alternative is the only other type of multipart supported now.
+          */
+          multipart = [bodyStructureKey objectForKey: @"multipart"];
+          if ([multipart isEqualToString: @""])
+            mixedPart = NO;
+          else
+            {
```

Comment:
why use brackets on `else` statement? When (1) first part (`if`) doesn't have them and (2) is only a single assignment. I don't have a strong opinion but either use brackets for both (`if` and `else`) or don't (on both). In this case I'd not use them, but as you wish.